kudos-webapps/src/main/webapp/vue-app/kudos/components/KudosApp.vue

```diff
@@ -177,20 +170,25 @@ import {initSettings} from '../../js/KudosSettings.js';
 export default {
   data() {
     return {
-      dialog: false,
+      drawer: false,
       listDialog: false,
+      placeholder: 'Write a message here',
```

Comment:
This should be internationalized. And there is already a label for it `exoplatform.kudos.label.kudosMessagePlaceholder` you can modify it.